docs/workshop.md

```diff
@@ -1223,12 +1225,43 @@ In this lab, you learned how to process and fetch orders using Azure Functions a
 
 # Lab 3 : Exposing and monetizing APIs (45m)
 
-For this Lab, we will focus on the following scope :
+For this Lab, we will focus on exposing and monetizing APIs using Azure API Management (APIM). The steps will guides you through the process of creating and managing APIs, setting up products, and securing the APIs using subscription keys and OAuth 2.0. The goal is to demonstrate how to publish APIs, manage access, and define usage plans to monetize the APIs effectively within the the following scope :
 
 ![Architecture diagram lab 3](./assets/lab3/architecture-schema-lab3.png)
 
+## About Azure API Management
+
+Azure API Management (APIM) is a fully managed service that enables organizations to publish, secure, transform, maintain, and monitor APIs. It provides a comprehensive set of features to manage the entire API lifecycle, from creation to retirement.
+
+Key features of APIM include:
+
+**API Gateway**: Acts as a facade for backend services, providing a single entry point for API consumers.
+Developer Portal: A customizable portal where developers can discover, learn about, and consume APIs.
+
+**Security**: Supports various authentication and authorization mechanisms, including subscription keys, OAuth 2.0, and JWT validation.
+
+**Rate Limiting and Throttling**: Controls API usage to prevent abuse and ensure fair usage.
+
+**Analytics and Monitoring**: Provides insights into API usage, performance, and health.
+
+**Transformation and Enrichment**: Allows modification of requests and responses, such as format conversion and data enrichment.
+
+**Policy Management**: Enables the application of policies to APIs for security, caching, transformation, and more.
+
+<div class="info" data-title="Note">
+
+> You can find a detailed article how APIM helps organizations expose their services as APIs, manage access, enforce policies, and gain insights into API usage, ultimately enabling them to monetize their APIs effectively. [following this link](https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts).
```

Comment:
There are no mentions to monetization in this article, it is the overview page which introduces the basic concepts of APIM :-(

Reply:
![image](https://github.com/user-attachments/assets/b3a1d268-a753-494e-975b-015c59062476)
a la fin du lab 3

Reply:
I meant the article https://learn.microsoft.com/en-us/azure/api-management/api-management-key-concepts in the paragraph https://github.com/ikhemissi/azure-ipaas-workshop/pull/27/files#diff-d88cdc3661331e4b78f6f54532457d2116e62ae6a491f1750818b6612dc0a6bcR1238